src/rows.ts

```diff
@@ -126,6 +148,17 @@ export class Rows {
     return columns;
   }
 
+  /**
+   * Rows.asObjects
+   * 
+   * Call this if you need to ouput the rows as objects.
+   * 
+   *     const rows = [...db.query("SELECT name FROM users;").asObjects()];
+   */
+  asObjects<T extends any = Record<string, any>>(): RowObjects<T> {
```

Comment:
I think the type annotation should live in `RowObjects` and shouldn't be generic (as we always return the same: an iterator over `Record`s)